instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -86,7 +86,18 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 			opts = append(opts, oteltrace.WithAttributes(rAttr))
 		}
 		ctx, span := tracer.Start(ctx, spanName, opts...)
-		defer span.End()
+		defer func() {
+			if cfg.RecordPanicConfig.enabled {
```

Comment:
note: added this as an opt-in behavior to preserve existing behavior

Reply:
I'm not sure we need this option. This is new behavior, not a breaking change.